src/platform/plugins/shared/navigation/server/ui_settings.ts

```diff
@@ -39,8 +40,9 @@ export const getUiSettings = (
           if (!request.auth.isAuthenticated) return DEFAULT_ROUTES.classic;
 
           const activeSpace = await spaces.spacesService.getActiveSpace(request);
+          const serverlessProjectType = setupPlugins.cloud?.serverless?.projectType;
 
-          const solution = activeSpace?.solution ?? 'classic';
+          const solution = serverlessProjectType ?? activeSpace?.solution ?? 'classic';
```

Comment:
> [!NOTE]
When Kibana is deployed for a serverless project, use the project type to attempt to access the default page.